src/asphalt/core/_context.py

```diff
@@ -636,9 +669,9 @@ async def start_background_task(
 
         :param func: the coroutine function to run
         :param name: descriptive name for the task
-        :param cancel_on_exit: whether to cancel the task when the context is exited
-            (``False`` means that the task will be allowed to run to its completion
-            before the context can finish)
+        :param teardown_action: the action to take when the context is being shut down:
+            ``None`` means no action, ``'cancel'`` means cancel the task, or you can
+            supply a callback that is called instead
```

Comment:
Should there be a mention that the teardown action can be async?

Reply:
Yes, that actually crossed my mind. I'll do that now.